packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -181,6 +181,17 @@ export function createKusama (t: TFunction): EndpointOption {
           OnFinality: 'wss://rpc.api.kico.dico.io'
         }
       },
+      {
```

Comment:
Ordering is where CI fails here - move this alphabetical.

Reply:
I'm so sorry, it is my wrong. I've just modified it.